### PR TITLE
adding the layout for product image, tags, ingredients, adding functi…

### PR DESCRIPTION
**Pull request by kristinadurant**
closing issue #18

### CODE REVIEW

File: client/src/components/ProductPage.jsx [hunk 1]
```diff
@@ -9,6 +9,8 @@ import Reviews from './Reviews';
 const ProductPage = () => {
   let { id } = useParams();
   const [product, setProduct] = useState({});
+  const [tab1, setTab1] = useState(true);
+  console.log(tab1);
```
Comment (Stommy123) [inline client/src/components/ProductPage.jsx:13]: could we remove this console log please?

File: client/src/components/ProductPage.jsx [hunk 2]
```diff
@@ -22,27 +24,45 @@ const ProductPage = () => {
...
-      <ProductImage image={product.image} verified={product.verified} />
-      <div className="tags">
-        {console.log(product)}
-        {product.tags?.map((tag, index) => {
-          return <img key={index} src={require(`../images/${tag}.svg`)} />;
-        })}
+      <div className="productContainer">
+        <ProductImage image={product.image} verified={product.verified} />
+        <div className="tags">
+          {console.log(product)}
```
Comment (Stommy123) [inline client/src/components/ProductPage.jsx:33]: same here

File: client/src/components/Reviews.jsx [hunk 3]
```diff
@@ -1,7 +1,50 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import axios from 'axios';
 
 const Reviews = () => {
-  return <div>Reviews</div>;
+  const [reviews, setReviews] = useState([]);
+  let { id } = useParams();
```
Comment (Stommy123) [inline client/src/components/Reviews.jsx:7]: could prolly be a `const` since we can't re-assign thisa

**Review by Stommy123 — APPROVED** [review]
LGTM, couple small non blocking comments

**Review by Amandat97 — APPROVED** [review]

**Pull request merged**